### ISSUE TITLE
Changing Color and Font

### ISSUE THREAD

**Issue by chasekregor** [chasekregor/chasekregor.github.io-source#2]
@jakevdp I wanted to thank you so much again for all your help with chasekregor.github.io

I was wondering where in the theme folder I can change the color at the top and the font. I don't really want my website to be a carbon copy of your site. I am not the best with html and css but have some familiarity. 

I went through some of your first commits to jakevdp.github.io-source but wasn't entirely able to find the one where you changed the theme color and font. 

Any help would be greatly appreciated. 

Once I set everything up I am going to start putting content up there. Just want to make it look good before I start putting stuff up there and showing it to people ;)

**Comment by jakevdp** [comment]
You have to do it in the CSS inside the ``theme`` directory, e.g. here: https://github.com/chasekregor/chasekregor.github.io-source/blob/master/theme/templates/main.css

I find it easiest to use Chrome developer tools (right click Inspect Element) to trace where font colors and other style things are coming from.

**Comment by chasekregor** [comment]
Yeah, the inspector tool is great. I need to learn more about using it. Thanks so much for pointing out the specific file I need to modify that helps a lot.
